src/drivers/cmakeDriver.ts

```diff
@@ -1716,9 +1712,9 @@ export abstract class CMakeDriver implements vscode.Disposable {
                 }
             } else {
                 if (gen) {
-                    if (/(Unix|MinGW) Makefiles|Ninja/.test(gen) && targets.length === 1 && targets[0] !== 'clean') {
+                    if (/(Unix|MinGW) Makefiles|Ninja/.test(gen) && (targets.length === 1 && targets[0] === 'clean')) {
```

Comment:
why is this condition changing?  This and the next change look incorrect.

Reply:
I originally change this because of a warning. Seems I made a mistake here, please revert those two lines as I am on my flight now

Reply:
Thanks. I reverted it.